src/arch/helperrvv.h

```diff
@@ -71,6 +71,11 @@
 #endif
 
 #if __riscv_v_intrinsic <= 12000
+// __riscv_vcreate* intrinsics only showed up fairly recently, but are useful
```

Comment:
Can we give be a little bit more specific than "fairly recently"?

Reply:
I've updated the comment but it turns out I'll need to update the logic as well.

It turns out it was only a coincidence that the feature added at the same time as the feature test macro changed in the compiler versions I checked.  Clang took a double-step, updating to 0.12 and then progresively adding the draft features without a version bump.  GCC is stuck in between, advertising 0.12 but not implementing vcreate (which is correct, I am wrong).

I'll submit a separate patch to test for clang-18 and up, as well as intrinsics version 1.0 and up.